<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MEMO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сыграть</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_theme">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Цветы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Животные</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Размер поля:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_theme">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тема:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_size">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>3 х 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 х 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 х 6</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Играть одному</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Играть вдвоем</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>Новая игра</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Об игре</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
  <action name="about_game">
   <property name="text">
    <string>Об игре</string>
   </property>
  </action>
  <action name="about_app">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
